main.go

```diff
@@ -51,6 +52,9 @@ func main() {
 		}
 	}
 
+	sha, release := version.GetReleaseInfo()
+	log.Printf("Starting Queue worker.\nVersion: %s\nGit Commit: %s", release, sha)
```

Comment:
queue-worker

Reply:
Please can you take the newlines out? Are they in the faas-netes controller?

Reply:
no, they are not in  faas-netes controller and the newlines are removed.